fdb-record-layer-lucene/docs/index.md

```diff
@@ -0,0 +1,15 @@
+# FoundationDB Record Layer Lucene Index Documentation
+
+## Data Layout
```

Comment:
What were you thinking here?  This will actually be a an "Index" in our current structure.  In some sense, this whole structure is an index.  Would love to understand better.

Reply:
My question was whether there would be another (record layer) index for fast lookup of items within FDB (e.g. if we would need to access a random file by name, or a random block, etc.) - it could be that looking up by primary key here is fine, I am just not aware of all the different ways in which these records can be looked up.
Thinking of Lucene Text Index as an "application", its queries may need index in its schema to run efficiently.

Reply:
The data model is structured so the access paths are effectively indexed.